README.md

```diff
@@ -1,69 +0,0 @@
-
-# Rime Python Voice Agent
```

Comment:
lets keep this main readme for the repo to orient users for the two different sub dirs. 

however each subdir can then have it's own Readme instructing how to use that particular code.